package.json

```diff
@@ -65,22 +69,24 @@
     "@types/lodash": "4.17.4",
     "@types/mapbox__mapbox-gl-draw": "^1.4.6",
     "ajv": "8.12.0",
+    "babel-plugin-react-compiler": "0.0.0-experimental-592953e-20240517",
     "blitz": "2.0.9",
     "dompurify": "^3.1.4",
+    "eslint-plugin-react-compiler": "0.0.0-experimental-c8b3f72-20240517",
     "immer": "10.1.1",
     "js-cookie": "3.0.5",
     "jsts": "2.11.3",
     "jsurl2": "2.2.0",
     "lodash": "4.17.21",
     "maplibre-gl": "4.3.2",
-    "next": "14.2.3",
+    "next": "14.3.0-canary.80",
     "next-auth": "4.24.7",
     "nuqs": "1.17.4",
     "pluralize": "8.0.0",
     "pmtiles": "3.0.5",
     "prisma": "5.14.0",
-    "react": "18.3.1",
-    "react-dom": "18.3.1",
+    "react": "19.0.0-rc-4c2e457c7c-20240522",
+    "react-dom": "19.0.0-rc-4c2e457c7c-20240522",
```

Comment:
"rc" , maybe?

Reply:
I did `npm install react@rc react-dom@rc types-react@rc types-react-dom@rc` and that is what came of that, so I think that is fine.

Reply:
Yeah, I just ran those installs again and it just updates those strings.